src/services/verifyToken.js

```diff
@@ -0,0 +1,24 @@
+/*
+Middleware to verify the JSON Web Token and validate the client
+*/
+import jsonwebtoken from 'jsonwebtoken';
+
+export default function verifyToken (req, res, next) {
+// Get the token from the HTTP headers
+  const SECRET_KEY = process.env.TOKEN_SIGNATURE || 'ssshhh';
+  var token = req.headers['x-json-web-token'];
```

Comment:
Can you switch this variable declaration to `const` or `let` so its consistent with the other code in this project